src/Components/CommentList.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import { Grid } from "@mui/material";
+import CommentCard from "./CommentCard";
+import { getCommentsById } from "../api";
+
+const CommentList = ({ review_id }) => {
+  const [commentList, setCommentList] = useState([]);
+
+  useEffect(() => {
+    getCommentsById(review_id).then((comments) => {
+      setCommentList(comments);
+    });
+  });
+
+  return (
```

Comment:
this is really nice, it would be great if you also had a message to the user if there's no comments on a review